Cargo.toml

```diff
@@ -5,8 +5,8 @@ members = ["devhub_common", "discussions", "community", "community-factory"]
 repository = "https://github.com/NEAR-DevHub/neardevhub-contract"
 
 [workspace.dependencies]
-near-sdk = "5.5.0"
-near-contract-standards = "5.5.0"
+near-sdk = "5.7.0"
+near-contract-standards = "5.7.0"
 devhub_common = { path = "./devhub_common" }
 cargo-near-build = "0.1.0"
```

Comment:
workspace version needs a bump to latest, as deployed subcontract won't work on 1.84

Reply:
this branch seems to be following a similar path, albeit a different destination https://github.com/dj8yfo/neardevhub-contract/tree/test_workspaces_0.17_precondition .
it doesn't have `.github/workflows/main.yml` and `.github/workflows/release.yml` updates